hedera-node/hedera-app-spi/build.gradle.kts

```diff
@@ -29,7 +29,5 @@ testModuleInfo {
     requires("org.assertj.core")
     requires("org.junit.jupiter.api")
     requires("org.junit.jupiter.params")
-    requires("org.mockito")
-    requires("org.mockito.junit.jupiter")
```

Comment:
I got an error on the **Module Info** GitHub check:
```
* What went wrong:
Execution failed for task ':app-spi:checkModuleDirectivesScope'.
> /home/runner/_work/hedera-services/hedera-services/hedera-node/hedera-app-spi/build.gradle.kts
  
  Please remove the following requires directives (or change to runtimeOnly):
    testModuleInfo {
      requires("org.mockito")
      requires("org.mockito.junit.jupiter")
    }

```

I'm not 100% sure why I got it but my hypothesis is that someone has deleted a code that was depending on these packages and pushed the PR with a failing **Module Info** step. Because I've made some changes in the **spi** package GitHub marked that as a problem on my PR.

Either way, it seems that we do not require these packages anymore so I deleted them.

Reply:
This may no longer be an issue since we reverted a gradle action version roll that went in yesterday morning.

Reply:
Yep, I don't see it in the diff anymore. Thanks :)